### PR TITLE
this is what I had to do to get this to compile

### PR DESCRIPTION
**Pull request by beanumber**
a few minor tweaks

### CODE REVIEW

**Comment by mine-cetinkaya-rundel** [comment]
It is odd, it compiled for me yesterday. Regardless, this is the master branch, so you did want to update the master branch for just completeness purposes? This is supposed to be Ed1 files, not the work-in-progress Ed2 files. Just checking before I merge.

**Comment by beanumber** [comment]
I guess? To be clear these changes are not substantive -- they were just necessary for me to compile. 

Can you compile without them?

File: makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+all: isrs.pdf
```
Comment (siddharthist) [inline makefile]: By convention, `Makefile` is usually capitalized.

**Comment by mine-cetinkaya-rundel** [comment]
I'm sorry that this just sat here as we moved focus at that time to getting the courses ready, and now there's a whole new structure so the PR becomes irrelevant. Thanks anyway though!

**Pull request closed without merging**